rust/core-lib/src/tabs.rs

```diff
@@ -807,19 +815,25 @@ impl CoreState {
 
     /// Handles changes in theme files.
     #[cfg(feature = "notify")]
-    fn handle_themes_fs_event(&mut self, event: DebouncedEvent) {
-        use self::DebouncedEvent::*;
-        match event {
-            Create(ref path) | Write(ref path) => self.load_theme_file(path),
+    fn handle_themes_fs_event(&mut self, event: Event) {
+        use notify::event::*;
+        match event.kind {
+            EventKind::Create(CreateKind::Any) | EventKind::Modify(ModifyKind::Any) => {
```

Comment:
I don't think we need to match `ModifyKind::Metadata` here, do we?

Reply:
We dont and it isnt. It's handled below in same method.